.github/workflows/robot/internal/bot/dismiss.go

```diff
@@ -125,8 +125,10 @@ func (b *Bot) deleteRuns(ctx context.Context, organization string, repository st
 			repository,
 			run.ID)
 		if err != nil {
-			return trace.Wrap(err)
+			log.Printf("Dismiss: Failed to dismiss workflow run %v: %v.", run.ID, err)
```

Comment:
Should this have a return after? Otherwise it'll print "successfully deleted" afterwards too?